### DISCUSSION TITLE
能否支持在线编辑hosts

### DISCUSSION THREAD

**Discussion by luodaoyi** [zdz/ServerStatus-Rust#28] [Ideas]
频繁买小鸡
然后每次加新机器都得去编辑配置文件.
包裹部署到railway上也是 每次要去编辑环境变量
能否做一个编辑页面 可以在线编辑hosts 重新写入到配置文件内
railway的话 可以考虑引入个数据库之类的

至于登录帐号 我觉得可以参考nezha监控的直接用github之类的授权

**Comment by zdz** [comment]
1.可以在配置文件预留一些主机配置，这样就不需要频繁动配置文件了，vps一般情况下都不会频繁动，续费半年或一年一次
2.引入db需要考虑一下，探针本来应该小而轻，引入db会带来部署复杂化，还有版本升级带来的数据库备份迁移兼容之类一系列的问题，一个配置文件多简单
3. 一个简单的探针需要github oauth授权吗？不是很同意
